docs/Use-Case-Cyclic-Voltammetry.md

```diff
@@ -0,0 +1,20 @@
+Use Case – cyclic voltammetry experiment for three electrode cell
+
+1. Input Initial Potential (V vs. Eocv)
```

Comment:
This breaks the UC rule that each statement must start with an actor.  You have to first identify an actor (who is your target audience) and then have he/she do the action.  It can be an abstract person, such as "student" but it is actually much better to have a clear idea of someone in mind who exemplifies your target audience.  You would write different code (or at least different documentation)  if the UC was for an undergrad in Uganda to use vs. a senior grad student who knew a lot of python.